packages/jest-config/src/normalize.ts

```diff
@@ -947,6 +947,29 @@ export default function normalize(
     newOptions.onlyChanged = newOptions.watch;
   }
 
+  if (!newOptions.onlyChanged) {
+    newOptions.onlyChanged = false;
+  }
+
+  if (!newOptions.lastCommit) {
+    newOptions.lastCommit = false;
+  }
+
+  if (!newOptions.onlyFailures) {
+    newOptions.onlyFailures = false;
+  }
+
+  if (!newOptions.watchAll) {
+    newOptions.watchAll = false;
```

Comment:
how about passing these through `Boolean` in switch/case above?

Reply:
the switch case only goes over the keys already there, so it might skip some keys? not sure

Reply:
I also want some to run after e.g. the coverage thing that happens after the `reduce`